.github/workflows/dev-deploy.yaml

```
@@ -0,0 +1,248 @@
+name: Build, Test, and Deploy Cloud Function, Run, and Workflow
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)